src/main/java/seedu/address/model/person/passenger/Passenger.java

```diff
@@ -75,19 +75,27 @@ public TripDay getTripDay() {
         return tripDay;
     }
 
+    public String getTripDayAsStr() {
+        return tripDay.toString();
+    }
+
     public TripTime getTripTime() {
         return tripTime;
     }
 
+    public String getTripTimeAsStr() {
+        return tripTime.toString();
```

Comment:
is this necessary? thinking because you could just use TripTime's internal toString() method implicitly but it could also be that im not too familiar with this

Reply:
Thanks for bringing this up. I think it might be useful as it would prevent the violation of the Law of Demeter when other classes try to call for the TripTime strings.